R/rdeephaven/R/client_options_wrapper.R

```diff
@@ -17,39 +17,65 @@
 #' want start a console upon client connection, you must ensure that the server you're connecting to was
```

Comment:
A general comment to apply to all files... "we" and "you" really bother some people in technical documentation, especially reference documentation.  @margaretkennedy should weigh in.

Reply:
We do not use the first person or contractions in our reference documentation. In our how-to guides and blogs, we do favor a more conversational tone.

Reply:
Combed through and removed first person. I believe I caught all contractions as well, but another pass would be good.